src/tests/api/BavHappyPathTest.test.ts

```diff
@@ -15,18 +15,21 @@ describe("Test BAV End Points", ()=> {
         //Session Request
         const stubResponse = await startStubServiceAndReturnSessionId(bavStubPayload);
         sessionId = stubResponse.data.session_id;
+        console.log("Session Id: " + sessionId);
```

Comment:
Still needed?

Reply:
Not essential, I like to have it in just incase we need to debug but happy to remove and we can always add for local runs when we needs it 👍